Makefile

```diff
@@ -29,6 +30,7 @@ docker:
 		.
 
 test:
+	go mod tidy
```

Comment:
Does this kind of duplicate to `./bin/test-go-mod-tidy.sh` below ?

Reply:
Humm, sort of, but the skip only checks to male sure it is tidy. We should evaluate if we need the script going forward.

Reply:
in this case I don't think the script is necessary .